graphql/testdata/TestTextMarshaling.snapshots.json

```diff
@@ -55,6 +87,14 @@
       }
     ]
   },
+  {
+    "Name": "invalid uuid func response",
```

Comment:
why don't either of these have batch executor in the name?

Reply:
Whoops, wasn't adding the prefix for error recordings, fixed.